code/game/objects/items/radio/headset.dm

```diff
@@ -302,10 +302,11 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 		usr << browse(null, "window=radio")
 
 
-/obj/item/radio/headset/mainship/mt
+/obj/item/radio/headset/mainship/st
 	name = "engineering radio headset"
 	icon_state = "eng_headset"
 	keyslot = new /obj/item/encryptionkey/engi
+	keyslot2 = new /obj/item/encryptionkey/req
```

Comment:
Load these during Initialize() to improve performance. Poke me on discord if you don't know how